src/clients/api/queries/useGetVoteDelegates.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery, QueryObserverOptions } from 'react-query';
+
+import { getVoteDelegates, GetVoteDelegatesOutput } from 'clients/api';
+import FunctionKey from 'constants/functionKey';
+import { useXvsVaultProxyContract } from 'clients/contracts/hooks';
+
+type Options = QueryObserverOptions<
+  GetVoteDelegatesOutput,
+  Error,
+  GetVoteDelegatesOutput,
+  GetVoteDelegatesOutput,
+  [FunctionKey.GET_VOTE_DELEGATES, string]
+>;
+
+const useGetVoteDelegates = ({ accountAddress }: { accountAddress: string }, options?: Options) => {
```

Comment:
This function seems to return the address the votes has been delegated to.
I'd make it extra that's what this is and change its name to `useGetVoteDelegateAddress`